### DISCUSSION TITLE
Let's talk about Remote Config

### DISCUSSION THREAD

**Discussion by Meldiron** [appwrite/appwrite#2389] [Ideas]
![GitHub Topics (2)](https://user-images.githubusercontent.com/19310830/140070522-172b4f53-1611-4017-bbe6-ec86345b7f70.png)

Appwrite's database is really powerful, but creating any functionality requires code from the developer's side that can be time-consuming. The idea of Remote Config is a special section inside Appwrite Console (collection under-the-hood) that should store the configuration of an Application, such as version, log, alert or how many items to load per page.

This feature could also support beta testing specific features by having a separate configuration for a specifically selected group of users.

---

Let's Chat! How can we implement Remote Config into Appwrite? What do you think about this initial feature? Can you see any problems or do you have an idea how to improve it even more?

**Comment by neiljaywarner** [comment]
@Meldiron  any update?  Firebase remote config *and* split.io both not working in china make me curious about this.

**Comment by stnguyen90** [comment]
Related issue: https://github.com/appwrite/appwrite/issues/323

**Comment by rafagazani** [comment]
>(collection under-the-hood) that should store the configuration of an Application, such as version, log, alert or how many items to load per page.

all these features is not already possible to achieve with a collection + realtime?
